packages/chat-client/src/components/ChatWidget/ChatBody.tsx

```diff
@@ -0,0 +1,200 @@
+import { DeepChat } from "deep-chat-react";
+import { useChat } from '../../contexts/ChatContext';
+import { Box, Typography, Paper } from '@mui/material';
+
+export const ChatBody = () => {
+  const { chatRef, modelId, chatConfig, isEmbedded } = useChat();
+
+
+  // Define custom styles for DeepChat
+  const deepChatStyles = {
+    messageStyles: {
+      default: {
+        user: {
+          bubble: {
+            backgroundColor: "#ff2020"
+          }
+        }
+      }
+    },
+    chatStyle: {
+      backgroundColor: chatConfig.theme?.background || '#ffffff',
+      height: '100%',
+      width: '100%'
+    },
+    inputAreaStyle: {
+      backgroundColor: chatConfig.theme?.secondary || '#f5f5f5',
+      borderTop: '1px solid #e0e0e0',
+      padding: '16px'
+    },
+    textInput: {
+      styles: {
+        container: {
+          backgroundColor: '#ffffff',
+          border: '1px solid #e0e0e0',
+          borderRadius: '24px',
+          padding: '12px 16px'
+        },
+        focus: {
+          border: `1px solid ${chatConfig.theme?.primary || '#d32f2f'}`,
+          boxShadow: `0 0 0 2px ${chatConfig.theme?.primary || '#d32f2f'}20`
+        },
+        text: { fontSize: '16px' }
+      },
+      placeholder: {
+        text: chatConfig.placeholder || "Type your message here...",
+        style: { color: '#757575' }
+      }
+    },
+    submitButtonStyles: {
+      submit: {
+        container: {
+          default: {
+            backgroundColor: 'transparent',
+            marginLeft: '8px',
+            marginRight: '8px'
+          },
+          hover: { backgroundColor: 'transparent' },
+          click: { backgroundColor: 'transparent' }
+        },
+        svg: {
+          styles: {
+            default: {
+              filter: 'brightness(0) saturate(100%) invert(49%) sepia(99%) saturate(477%) hue-rotate(343deg) brightness(100%) contrast(101%)'
+            }
+          }
+        }
+      }
+    },
+    auxiliaryStyle: `
+      ::-webkit-scrollbar {
```

Comment:
Do we really need to override the scrollbar style? Also having this as a massive string is really odd and would be better in a CSS document